### PR TITLE
Add actuals

### PR DESCRIPTION
**Pull request by lxu213**

### CODE REVIEW

**Comment by BenjiLee** [comment]
400k lines  >.<

File: src/parser/actual.js [hunk 1]
```diff
@@ -1,80 +1,113 @@
...
+    '36019', '36021', '36023', '36025', '36027', '36029', '36031', '36033', '36035', 
+    '36037', '36039', '36041', '36043', '36045', '36047', '36049', '36051', '36053', 
+    '36055', '36057', '36059', '36061', '36063', '36065', '36067', '36069', '36071', 
+    '36073', '36075', '36077', '36079', '36081', '36083', '36085', '36087', '36089', 
+    '36091', '36093', '36095', '36097', '36099', '36101', '36103', '36105', '36107', 
+    '36109', '36111', '36113', '36115', '36117', '36119', '36121', '36123']
+const params = ['deaths'];
+const states = ['06', '36'];
+
+const path = '/Users/lxu213/Documents/covid-dashboard-app/src/store/USAFacts.csv';
```
Comment (BenjiLee) [inline src/parser/actual.js:103]: this path is hardcoded to your computer

File: src/parser/actual.js [hunk 2]
```diff
@@ -1,80 +1,113 @@
...
-getActualData()
-createNYTmapping()
+module.exports = {
+    initActualObj,
+    readActual,
+    aggregateActualByState
+};
+
+// all CA and NY counties 
+const geoids = [
```
Comment (BenjiLee) [inline src/parser/actual.js:85]: A better way to name/separate this is to have three different const. One for CA and the other for NY, and then another const that collects the counties. The code then becomes self documenting.

Example:
```
const countyNY = [...]
const countyCA = [...]
// Pick whatever array merging method you want here.
const allCounties = [].concat(countyNY, countyCA);
```

File: src/parser/actual.js (hunk 1 above)
Comment (lxu213) [reply src/parser/actual.js:103]: Oh right this is temporary until it can be brought into another pipeline - is it bad that its 400k line?

Comment (BenjiLee) [reply src/parser/actual.js:103]: Nothing wrong with 400k lines. Just makes the PR lines of code diff look funny

**Pull request merged**